tgui/packages/tgui/components/TextArea.tsx

```diff
@@ -67,7 +70,7 @@ export const TextArea = forwardRef(
     const handleKeyDown = (event: KeyboardEvent<HTMLTextAreaElement>) => {
       if (event.key === KEY.Enter) {
         if (event.shiftKey) {
-          onChange?.(event as any, event.currentTarget.value);
+          event.currentTarget.focus();
```

Comment:
why is this behavior changing? it should keep onInput but does it need to focus? the idea is that it starts a new line